photon-client/src/main/scala/com/linkedin/photon/ml/hyperparameter/tuner/AtlasTuner.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.hyperparameter.tuner
+
+import org.apache.spark.sql.DataFrame
+
+import com.linkedin.photon.ml.HyperparameterTuningMode
+import com.linkedin.photon.ml.cli.game.training.GameTrainingDriver.{getOrDefault, hyperParameterTuning, hyperParameterTuningIter}
+import com.linkedin.photon.ml.estimators.{GameEstimator, GameEstimatorEvaluationFunction}
+import com.linkedin.photon.ml.estimators.GameEstimator.GameResult
+import com.linkedin.photon.ml.hyperparameter.search.{GaussianProcessSearch, RandomSearch}
```

Comment:
Related to the above comment, in that setup, we will move out all functions from hyperparameter correct?

Reply:
This tuner class implementing `runHyperparameterTuning` will be unique for different models, e.g. GAME and XGBoost since they have their own customized `runHyperparameterTuning`. It is like an integration point which I prefer to keep it in photon-client and xgboost. We are not moving all the functions from `hyperparameter` folder out. For example, `GameHyperparameterDefaults` will stay too.